.github/workflows/on_prerelease.yaml

```diff
@@ -9,13 +9,14 @@ on:
 
 jobs:
   pre-release:
-    uses: newrelic/coreint-automation/.github/workflows/reusable_pre_release.yaml@v3
+    uses: newrelic/coreint-automation/.github/workflows/reusable_pre_release.yaml@v4
     with:
```

Comment:
Nice, and I could see the new version worked as expected.

Reply:
Yeah, it worked. Thanks.